### DISCUSSION TITLE
Welcome to terraform-provider-confluence Discussions!

### DISCUSSION THREAD

**Discussion by mirogta** [mirogta/terraform-provider-confluence#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mirogta** [comment]
Enabling "new editor experience", aka editor v2:

- https://community.atlassian.com/t5/Confluence-questions/Enable-new-editor-for-pages-added-through-REST-API/qaq-p/1477533
- https://jira.atlassian.com/browse/CONFCLOUD-68057

**Comment by DavidGamba** [comment]
Hi, glad to see someone is moving this plugin forward.
Will you enable the issues feature?

Thanks!

PS: I just saw the plugin, have been using the confluence APIs directly for a while but upgrading to Terraform will be nice.

**Reply by mirogta** [reply]
Issues enabled
